React-Native/TodoList/src/app/index.js

```diff
@@ -1,16 +1,14 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Provider } from 'react-redux';
 import store from '../redux/store';
 import AppContainer from './AppContainer';
 
-// import '../config';
-
-export default class App extends Component {
-  render () {
+const App = function App () {
```

Comment:
mmmm the function is not well define: delete the `const App =`, just function App () { ...